python/private/bzlmod/pip.bzl

```diff
@@ -714,7 +729,54 @@ extension.
 """,
         ),
     },
-    **_extension_extra_args()
+)
+
+pip_internal = module_extension(
```

Comment:
This needs to declare os/arch dependence.

Reply:
Not necessarily, since we are calling PyPI to get the right dependencies, the lock file will be the same on all platforms, so this can be removed, I'll change it so that it is the default behaviour.

Reply:
Nice!